components/range_cache_memory_engine/src/background.rs

```diff
@@ -1224,7 +1183,12 @@ pub struct DeleteRangeRunner {
 
 impl DeleteRangeRunner {
     fn new(engine: Arc<RwLock<RangeCacheMemoryEngineCore>>) -> Self {
+        let rt = runtime::Builder::new_current_thread()
```

Comment:
What's the purpose of introducing this async runtime, I don't see any benefits from the changes of this PR

Reply:
It is used to execute the callback.